src/main/java/io/cryostat/net/OpenShiftAuthManager.java

```diff
@@ -296,6 +309,29 @@ public AuthenticationScheme getScheme() {
         }
     }
 
+    private Future<Boolean> deleteToken() {
+        try (OpenShiftClient client = clientProvider.apply(getServiceAccountToken())) {
+            String serviceAccountAsOAuthClient = this.getServiceAccountName();
+            List<OAuthAccessToken> userOauthAccessTokens =
+                    client.oAuthAccessTokens().list().getItems().stream()
+                            .filter(t -> t.getClientName().equals(serviceAccountAsOAuthClient))
+                            .collect(Collectors.toList());
+
+            Boolean deleted =
+                    Optional.ofNullable(client.oAuthAccessTokens().delete(userOauthAccessTokens))
+                            .orElseThrow(TokenNotFoundException::new);
+
+            if (!deleted) {
```

Comment:
I think, since `deleted` is nullable, this would be safer as `Boolean.FALSE.equals(deleted)`